### ISSUE TITLE
How to generate video demos from your Code？

### ISSUE THREAD

**Issue by gd2016229035** [Yuanbo2020/Audio-Visual-VAD#1]
Thank you for your work! 
I wonder whether your source code only contains training and evaluation part？
Is there a public pretrained model? And how to generate video demos easily like your Video_demos part?

**Comment by Yuanbo2020** [comment]
@gd2016229035 Yes, the code is not complete, because following the corresponding rules of Bytedance during my internship, I have no right to publish the complete code and training data-related codes and models. This is a strict rule. I'm sorry.

Regarding video demos, firstly, we extract the audio and image in the video clip by FFmpeg, then, the detection result, the label text is added on images within the time range of the event occurrence. Finally, the original audio and the images with the tagged label text are synthesized into a video clip.

**Comment by gd2016229035** [comment]
Thank you for your kindly reply!~ 
I just want to comfirm：
Do you mean that even the training part is also not complete? I can not train model directly？
Only model structure details from source code and a part(MAVC100) of the MVD500 dataset can be referred to?

**Comment by Yuanbo2020** [comment]
Hi, in the training, there is no feature extraction-related codes, but the model part and the training processing are good. The flow is firstly you extract the feature from source video clips, then input them into the model and train, but the feature extraction part is related to the private dataset MVD500, so I have no right to upload it.

we used MVD500 to train the model, and MAVC100 was only used in the evaluation. The source video clips in MAVC100 are from the Internet and we labeled them, so I can publish it.
